pkgs/development/libraries/fltk/default.nix

```diff
@@ -39,10 +54,13 @@ composableDerivation.composableDerivation {} rec {
     localpngSupport = false;
     sharedSupport = true;
     threadsSupport = true;
-  };
+  } // cfg;
 
   meta = {
-    description = "a C++ cross platform lightweight gui library binding";
+    description = "A C++ cross-platform light-weight GUI library binding";
```

Comment:
Not sure it makes sense to call this a "binding", seems like "library" suffices.